### PR TITLE
test: Some PromQL cases about aggregator

### PR DESCRIPTION
**Pull request by waynexia**
I hereby agree to the terms of the [GreptimeDB CLA](https://gist.github.com/xtang/6378857777706e568c1949c7578592cc)

## What's changed and what's your intention?

Port some aggregator cases from https://github.com/prometheus/prometheus/blob/e40deb1086265f7912dea2d4b18a8faf35dfed7b/promql/testdata/aggregators.test

I plan to add others as sqlness cases in the future. It'd support promql in the near future 

## Checklist

- [ ]  I have written the necessary rustdoc comments.
- [x]  I have added the necessary unit tests and integration tests.

### CODE REVIEW

**Comment by waynexia** [comment]
>>Merging [#977](https://codecov.io/gh/GreptimeTeam/greptimedb/pull/977?src=pr&el=desc&utm_medium=referral&utm_source=github&utm_content=comment&utm_campaign=pr+comments&utm_term=GreptimeTeam) (https://github.com/GreptimeTeam/greptimedb/commit/597949ae7250bfeabd8dc0b4603c956b28d40336) into [develop](https://codecov.io/gh/GreptimeTeam/greptimedb/commit/4c925e007944144f55fe9c5291d926f4b2fce0ea?el=desc&utm_medium=referral&utm_source=github&utm_content=comment&utm_campaign=pr+comments&utm_term=GreptimeTeam) (https://github.com/GreptimeTeam/greptimedb/commit/4c925e007944144f55fe9c5291d926f4b2fce0ea) will decrease coverage by 0.11%.

🧐 why a test PR will decrease the coverage

File: src/datanode/src/instance/sql.rs [hunk 1]
```diff
@@ -165,8 +167,31 @@ impl Instance {
...
     }
 
-    pub async fn execute_promql(&self, sql: &str, query_ctx: QueryContextRef) -> Result<Output> {
-        let stmt = QueryLanguageParser::parse_promql(sql).context(ExecuteSqlSnafu)?;
+    pub async fn execute_promql(&self, promql: &str, query_ctx: QueryContextRef) -> Result<Output> {
+        let stmt = QueryLanguageParser::parse_promql(promql).context(ExecuteSqlSnafu)?;
+        self.execute_stmt(stmt, query_ctx).await
+    }
+
+    // TODO: merge this and `execute_promql` after #951 landed
```
Comment (killme2008) [inline src/datanode/src/instance/sql.rs:175]: `TODO(ruihang)`

File: src/datanode/src/sql/insert.rs [hunk 2]
```diff
@@ -43,7 +43,11 @@ impl SqlHandler {
             table: &req.table_name.to_string(),
         };
 
-        let table = self.get_table(&table_ref)?;
+        let table = self
+            .catalog_manager
+            .table(table_ref.catalog, table_ref.schema, table_ref.table)
+            .unwrap()
```
Comment (killme2008) [inline src/datanode/src/sql/insert.rs:49]: is the `unwrap` safe here?

Comment (waynexia) [reply src/datanode/src/sql/insert.rs:49]: Nice catch... 👍

File: src/datanode/src/tests/promql_test.rs [hunk 3]
```diff
@@ -37,31 +76,266 @@ async fn sql_insert_promql_query_ceil() {
...
+    ('api-server', '0', 'canary', 300, 0),
+    ('api-server', '1', 'canary', 400, 0),
+    ('app-server', '0', 'production', 500, 0),
+    ('app-server', '1', 'production', 600, 0),
+    ('app-server', '0', 'canary', 700, 0),
+    ('app-server', '1', 'canary', 800, 0);"#;
+
+fn unix_epoch_plus_50m() -> SystemTime {
+    UNIX_EPOCH.checked_add(Duration::from_secs(100)).unwrap()
+}
```
Comment (evenyag) [inline src/datanode/src/tests/promql_test.rs:129]: Isn't this `plus_100s`?

**Review by killme2008 — APPROVED** [review]
LGTM

**Review by evenyag — APPROVED** [review]
LGTM

**Pull request merged**